test/jdk/ProblemList.txt

```diff
@@ -540,6 +540,8 @@ javax/management/remote/mandatory/notif/NotifReconnectDeadlockTest.java 8042215
 
 # jdk_net
 
+java/net/HttpURLConnection/HttpURLConnectionExpectContinueTest.java 8326549 generic-all
```

Comment:
Please use [8326503](https://bugs.openjdk.org/browse/JDK-8326503) as the bug reference as `8326549` has been closed as a duplicate.

Reply:
After opening my bug, I had seen the one by Sendao. I linked the two with a "duplicates"  link, and asked Sendao to close one of them not knowing which one will make the race ... I think this was before you opened the sub-task. But all this is not a big deal, is it? After all, we now have a real fix!